packages/app-cli/tests/support/plugins/imaging/api/JoplinImaging.d.ts

```diff
@@ -1,12 +1,35 @@
 import { Rectangle } from './types';
-export interface Implementation {
-    nativeImage: any;
-}
 export interface CreateFromBufferOptions {
     width?: number;
     height?: number;
     scaleFactor?: number;
 }
+export interface CreateFromPdfOptions {
+    /**
+     * The first page to export. Defaults to `1`, the first page in
+     * the document.
+     */
+    minPage?: number;
+    /**
+     * The number of the last page to convert. Defaults to the last page
+     * if not given.
+     *
+     * If `maxPage` is greater than the number of pages in the PDF, all pages
+     * in the PDF will be converted to images.
+     */
+    maxPage?: number;
+    scaleFactor?: number;
+}
+export interface PdfInfo {
+    numPages: number;
```

Comment:
Could you name it "pageCount" please since this is what we use for pagination in various APIs? (often "page_count" actually but "pageCount" would make more sense in this context)